src/core/core.js

```diff
@@ -498,15 +497,13 @@ define(function (require) {
 
   p5.prototype.registerPreloadMethod = function(m) {
     p5.prototype._preloadMethods.push(m);
-  }.bind(this);
+  };
```

Comment:
We'll need somebody to weigh in on why these binds were here to begin with before we can tell if this is a safe change. Didn't break any tests, but we don't test everything :)